migrator/models.py

```diff
@@ -140,7 +140,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
     def parse(dirname: str) -> RevisionList:
         assert os.path.isdir(dirname)
         revisions = {}
-        for f in glob.glob(os.path.join(dirname, "*.yml")):
+        for f in sorted(glob.glob(os.path.join(dirname, "*.yml"))):
```

Comment:
I added this since there's a check (look line 131) that was failing for me when the files were not sorted